pkg/sql/set_zone_config.go

```diff
@@ -210,6 +210,10 @@ func (n *setZoneConfigNode) startExec(params runParams) error {
 			inheritVal, expr := val.inheritValue, val.explicitValue
 			if inheritVal {
 				copyFromParentList = append(copyFromParentList, *name)
+				if optionStr.Len() > 0 {
+					optionStr.WriteString(", ")
+				}
+				fmt.Fprintf(&optionStr, "%s = COPY FROM PARENT", string(*name))
```

Comment:
wait what no you should not convert the name to a string!!

it's `fmt("%s = ...", name)`